src/diffusers/training_utils.py

```diff
@@ -5,6 +5,7 @@
 
 import numpy as np
 import torch
+import transformers
```

Comment:
Actually `transformers` is still not a hard dependency (some training scripts don't need `transformers`)